src/main/java/no/nav/pto/veilarbportefolje/admin/AdminController.java

```diff
@@ -180,25 +173,33 @@ public String forceShardAssignment() {
         return opensearchAdminService.forceShardAssignment();
     }
 
+    @PostMapping("/pdl/lastInnDataFraPdl")
+    public String lastInnPDLBrukerData() {
+        authorizeAdmin();
+        AtomicInteger antall = new AtomicInteger(0);
+        List<AktorId> brukereUnderOppfolging = oppfolgingRepositoryV2.hentAlleGyldigeBrukereUnderOppfolging();
```

Comment:
this one is fetching specific number of rows since we are using `fetchSize` in `hentAlleGyldigeBrukereUnderOppfolging`